examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
There could be other places we've done the same thing. It would be good to fix them but probably in another PR. Yes notch all then proj is correct, and yes please note this in the BUG section.